fastlane/Fastfile

```diff
@@ -1,7 +1,9 @@
-# All lines starting with a # are ignored when running `fastlane`
-
 fastlane_version '2.83.0'
 
+before_all do |lane, options|
+  xcversion(version: '9.3')
```

Comment:
What is this for?

Reply:
It is requirement for current controls to be built only with Xcode 9.3